### PR TITLE
Add Support for Open Graph for ol.io

### PR DESCRIPTION
**Pull request by mbroz2**
#### What was fixed?  (Issue # or description of fix)
Adds support for Open Graph
#### Were the changes tested on
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [ X] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] Dymanic Accessability Plugin (DAP)
- [ ] Lighthouse (in Chrome dev tools)

### CODE REVIEW

**Comment by steven1046** [comment]
We probably want to add the same meta tags in the doc build in head-meta.hbs.

File: src/main/content/_includes/head.html [hunk 1]
```diff
@@ -47,6 +47,11 @@
   <meta name="twitter:title" content="{% if page.seo-title %}{{ page.seo-title }}{% else %}{{site.title}}{% endif %}" />
   <meta name="twitter:description" content="{{ description }}" />
   <meta name="twitter:image" content="https://openliberty.io/img/twitter_card.jpg" />
+
+  <meta property='og:title' content="{% if page.seo-title %}{{ page.seo-title }}{% else %}{{site.title}}{% endif %}" />
+  <meta property='og:image' content="{% if page.open-graph-image %} {{page.open-graph-image}} {% else %}https://openliberty.io/img/twitter_card.jpg{% endif %}" />
+  <meta property='og:description' content="{{ description }}" />
+  <meta property='og:url' content="{{ page.url | replace:'index.html','' | absolute_url }}" />
```
Comment (ellenwyllie) [inline src/main/content/_includes/head.html:54]: The opengraph website says 'type' is a required property so we might need to add that as well.

Comment (mbroz2) [reply src/main/content/_includes/head.html:54]: yes, not sure how I dropped that from the commit, will add it back along with the update to head-meta for docs

**Comment by mbroz2** [comment]
Example docs output after adding OG:
image

**Pull request merged**